queries/vue/injections.scm

```diff
@@ -1,4 +1,35 @@
-; inherits: html_tags
+(
+  (style_element
+    (start_tag) @_no_lang
+    (raw_text) @css)
+  (#not-contains? @_no_lang "lang=")
+) 
+
+(
+  (style_element
+    (start_tag
+      (attribute
+        (quoted_attribute_value (attribute_value) @_lang)))
+    (raw_text) @css)
+  (#eq? @_lang "css")
+)
+
+; if start_tag does not specify `lang="..."` then set it javascript
+(
+ (script_element
+    (start_tag) @_no_lang 
+  (raw_text) @javascript)
+ (#not-contains? @_no_lang "lang=")
+)
+
+(
+  (script_element
+    (start_tag
+      (attribute
+        (quoted_attribute_value (attribute_value) @_lang)))
+    (raw_text) @javascript)
+  (#any-of? @_lang "js" "javascript")
```

Comment:
Does it require case for `ts` as well or will it just work for `typescript`?

Reply:
> Does it require case for `ts` as well or will it just work for `typescript`?
This is based on the `ts` case below that, and it works well from what I experienced, so I do it for javascript as well

Reply:
Actually, when setting `lang` attribute to either `javascript` or `typescript`, volar will responds back with an error

@gegoune should i remove both it and typescript then

![image](https://user-images.githubusercontent.com/29790821/208237823-ec3e63c3-c28f-42fd-911c-585825d26cb9.png)

Reply:
`lang` should be `ts` or `js`, not full names.

My original comment was a mistake by not looking at entire file but only your changes - apologies for confusing you.

Reply:
> `lang` should be `ts` or `js`, not full names.
> 
> My original comment was a mistake by not looking at entire file but only your changes - apologies for confusing you.

Just removed it, can you have another check to see if there's issues?